test/api/README.md

```diff
@@ -0,0 +1,22 @@
+# Testing the T-Route FastAPI Extension
```

Comment:
Would like to ultimately see some automated unit tests of the API -- setup the instance with the test_compose inside a test framework's init/setup (e.g. pytest) and and create then send requests to the server and verify the expected return messages at least.